packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx

```diff
@@ -59,6 +59,10 @@ export interface ChatMessageActionFlyoutProps {
 export const ChatMessageActionFlyout = (props: ChatMessageActionFlyoutProps): JSX.Element => {
   const theme = useTheme();
   const messageReadByCount = props.messageReadBy?.length;
+
+  props.messageReadBy?.sort(function (a, b) {
```

Comment:
I don't love that this mutates the original array. If this array is used by another component it will be impacted by this.
consider sorting before passing in, or here do:
```ts
const sortedMessageReadyByList = [...props.messageReadBy].sort((a,b) => a.displayName.localeCompare(b.displayName));
```